tests/cases/fourslash/callHierarchyContainerName.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path="fourslash.ts" />
+
+////function /**/f() {}
+////
+////class A {
+////  static sameName() {
+////    f();
+////  }
+////}
+////
+////class B {
+////  sameName() {
+////    A.sameName();
+////  }
+////}
+////
+////const Obj = {
+////  get sameName() {
+////    return new B().sameName;
+////  }
+////};
+////
+////namespace Foo {
+////  function sameName() {
+////    return Obj.sameName;
+////  }
+////
+////  export class C {
+////    constructor() {
+////      sameName();
+////    }
+////  }
+////}
+////
+////module Bar {
```

Comment:
we also allow module Bar.Foo { } so need to handle that case as well ?

Reply:
IIRC the AST for that is a ModuleDeclaration with a ModuleDeclaration for a body (no qualified names anywhere), so should already be handled. A test might be relevant though.

Reply:
That raises a question for @mjbvz though about whether the `containerName` for declarations inside nested namespaces would ideally be just the innermost name (`Foo` in @sheetalkamat’s example) or the fully-qualified name (`Bar.Foo`).

Reply:
I feel it should match the document outline (which uses `navtree`), so it would be just `Foo` in this example